src/server/req_signal.c

```diff
@@ -548,6 +552,12 @@ create_resreleased(job *pjob)
 			free(resreleased);
 			return 1;
 		}
+		/* If there are no resources released on this vnode then add a dummy "ncpus=0"
+		 * This is needed otherwise scheduler will not be able to assign this chunk to
+		 * the job while trying to resume it
+		 */
+		if (no_res_rel)
+			strcat(resreleased,dflt_ncpus_rel);
```

Comment:
Add a space after the comma.  Also is it possible we won't have enough space?  We're allocating enough space for 1x the exec_vnode.  We're now adding something that may not have been in the exec_vnode to begin with.